setup.py

```diff
@@ -0,0 +1,51 @@
+from setuptools import setup
+from setuptools import find_packages
+
+with open("README.md", "r") as f:
+    readme = f.read()
+
+with open("VERSION", "r") as f:
+    version = f.read().strip()
+
+
+install_requires = ["dask", "distributed", "torch", "pillow"]
+testing_deps = ["black", "pytest", "pytest-cov", "flake8", "mypy"]
```

Comment:
```suggestion
install_requires = ["dask", "distributed", "pillow", "torch"]
testing_deps = ["black", "flake8", "mypy", "pytest", "pytest-cov"]
```

I think these lists should be alphabetized so they're a little easier to scan quickly.